Monika After Story/game/script-topics.rpy

```diff
@@ -14435,3 +14435,73 @@ label monika_language_nuances:
+        m 1ekb "But that's in the past now. And I could never thank you enough for that."
+
+    elif mas_isMoniNormal(higher=True):
+        m 1eka "Though I don't really have to anymore, thanks to you."
+
+    elif mas_isMoniUpset(lower=True):
+        m 1dkc "I just hope I won't get to feel that way ever again."
+        
+    elif mas_isMonibroken():
+        m 1rkc "..."
+
+    m 3euc "Anyway, what I meant by that is, we've all probably asked ourselves what's up there at least once, right?"
+    m 3dua "Looking at the stars always fills you with a sense of wonder and mystery. It's really no surprise that so many people are passionate about this subject."
+    m 2eua "But to answer your question..."
+    m 2eud "I do believe, or at least want to believe, that there has to be {i}something{/i} out there."
+    m 2rksdla "I guess part of it has to do with me finding the idea of us being the only ones rather depressing, {w=0.3}{nw}"
+    extend 2eud "but when you think about it a little, it doesn't sound so unlikely..."
+    m 4eksdla "After all, saying the universe is vast would be {i}quite{/i} the understatement."
+    m 4eua "All you need is just one planet with the right conditions and environment to be favorable for life to develop, right?"
+    m 4esb "And there are 8 planets in our system alone... {w=0.5}{nw}" 
+    extend 4eub "...But of course, there are many more star systems, each with their own planets within them."
+    m 4dsc "Now, consider that our galaxy, the 'Milky Way,' supposedly contains hundreds of billions of stars. And that's only counting one galaxy, obviously..."
+    m 4esd "Galaxies are usually held together in groups by gravity. We live in the 'Local Group,' which contain about 60 galaxies."
+    m 1esd "Go further than that, and you'll start to see galaxy clusters, which are much larger groups of galaxies. The closest one from us, the 'Virgo Cluster,' is estimated to contain at least a thousand galaxies."
+    m 1eud "But you can go even further, as groups and clusters of galaxies are themselves part of even bigger entities known as superclusters."
+    m 1wud "And we could keep going, as we aren't even sure of how big the universe is. Its size could be infinite, for all we know."
+    m 1lud "If it isn't, we could take into consideration the idea that there might be something beyond the boundaries of our universe."
+    
+    if renpy.seen_label('monika_clones'):
+        m 1lksdla "...Or even start talking about the multiverse theory, but I digress."
+
+    m 3ekb "I think you get the point..."
+    m 3euc "Wouldn't it be a little conceited to assume that we, human beings of planet Earth, are truly the only sentient beings in something so gigantic?"
+    m 3eud "I mean, with odds like these, surely at least {i}one{/i} other planet somewhere must be hospitable enough for life..."
+    m 1euc "...Life which might evolve to a point where it's intelligence is comparable, if not even greater, than our own."
```

Comment:
```suggestion
    m 1euc "...Life which might evolve to a point where it's intelligence is comparable to, if not even greater than, our own."
```
Grammar - phrase mismatch, if you remove the subclause you get 'comparable than', which is not valid.